src/styles/comp.scss

```diff
@@ -0,0 +1,73 @@
+.comp {
+  display: flex;
+  flex-direction: column;
+  margin-inline: $margin-inline;
+  font-family: $font-family;
```

Comment:
Ensure that the variables `$margin-inline`, `$font-family`, `$text-color-strong`, and `$text-color` are defined in a separate SCSS variables file. This is important for maintaining consistency across your styles and making updates easier.